cms/templates/js/certificate-details.underscore

```diff
@@ -0,0 +1,40 @@
+<div class="collection-details wrapper-certificate">
+    <header class="collection-header">
+        <h3 class="title">
+            <a href="#" class="toggle detail-toggle <% if (showDetails){ print('hide'); } else { print('show'); } %>-details">
+                <i class="ui-toggle-expansion icon fa fa-caret-<% if (showDetails){ print('down'); } else { print('right'); } %>"></i>
+                <%= name %>
+            </a>
+        </h3>
+    </header>
+
+   <ol class="collection-info certificate-info certificate-info-<% if(showDetails){ print('block'); } else { print('inline'); } %>">
+     <% if (!_.isUndefined(id)) { %>
+         <li class="certificate-id">
+            <span class="certificate-label"><%= gettext('ID') %>: </span>
+            <span class="certificate-value"><%= id %></span>
+        </li>
+     <% } %>
+     <% if (showDetails) { %>
+        <li class="collection-description certificate-description">
+            <%= description %>
+        </li>
+        <li class="collection-description certificate-description">
+         <header>
+           <label class="title-2"><%= gettext("Course Signatories") %></label>
+         </header>
+         <div class="signatory-details-list"></div>
+        </li>
+
+      <% } %>
+   </ol>
+
+    <ul class="actions certificate-actions">
```

Comment:
Is there a reason why this is an unordered list?  I'm not a fan of using ```<ul>``` or ```<ol>``` for anything but lists of items.  They add a lot of verbosity for screen reader users.  While they are useful for lists of items, they actually become cumbersome in other contexts.